index.bs

```diff
@@ -10119,6 +10186,15 @@ and its value is an object called the <dfn id="dfn-interface-object" export>inte
 The property has the attributes { \[[Writable]]: <emu-val>true</emu-val>, \[[Enumerable]]: <emu-val>false</emu-val>, \[[Configurable]]: <emu-val>true</emu-val> }.
 The characteristics of an interface object are described in [[#interface-object]].
 
+If the [{{LegacyWindowAlias}}] extended attribute was specified on an interface,
+then for each [=identifier=] mentioned in the extended attribute,
+there must be a corresponding property on the [=primary global interface=],
+known as the <dfn id="dfn-legacy-window-alias" export>legacy window alias</dfn>.
+The name of the property is the [=legacy window alias=]'s [=identifier=],
```

Comment:
What we did for NamedConstructor is define a "NamedConstructor identifier" in the [NamedConstructor] section above. Maybe you'd want something similar here, especially if we get rid of defining the [=legacy window alias=]?